### ISSUE TITLE
Update stack for safecastrpt-prd (https://grafana.safecast.cc/)

### ISSUE THREAD

**Issue by matschaffer** [Safecast/safecastapi#637]
Got an email from AWS that the stack safecastrpt-prd (https://grafana.safecast.cc/) is running on is no longer getting updates.

Should be able to copy the config and create a new environment on the latest stack.

Note that this stack is currently "64bit Amazon Linux 2018.03 v2.9.1 running PHP 5.6" but doesn't actually use PHP.

Just reached for that as a fairly generic stack with a webserver that provides virtualhost support.

Could probably use a variety of stacks, but watch out not to use nginx, or if you do make sure to update stuff like https://github.com/Safecast/reporting/blob/master/.ebextensions/grafana.config#L59-L66

Documenting some headway here:

- [x] Set up github action CI pipeline
- [x] Set up new docker-based dev env
- [x] Run dev env on spot instances
- [x] Use safecast_deploy for deployment
- [x] Ensure CI pipeline can work for PR forks
- [x] Move CI pipeline bundle step to https://github.com/Safecast/github-actions
- ~Try letsencrypt wildcard to avoid using LB in dev  (~$18/mo for elb)~
- ~Work out a nightly master grafana tag~

**Comment by matschaffer** [comment]
Posting these images for https://forums.aws.amazon.com/thread.jspa?messageID=952947&#952947 since AWS forum images uploads seem to be broken at the moment.

![Screen Shot 2020-08-19 at 10 58 40 AM](https://user-images.githubusercontent.com/690/90584057-abead400-e20c-11ea-9b58-9ad3293805ad.png)
![Screen Shot 2020-08-19 at 11 02 45 AM](https://user-images.githubusercontent.com/690/90584069-b0af8800-e20c-11ea-9447-eddaa1a50da9.png)
![Instances___EC2_Management_Console](https://user-images.githubusercontent.com/690/90584101-c624b200-e20c-11ea-9a3b-456189257069.png)

**Comment by matschaffer** [comment]
Looks like https://github.com/go-acme/lego might be a better option for the certs since it has route53 support directly. Will give it a shot.

**Comment by matschaffer** [comment]
Looks like beanstalk supports shared ALBs now so gonna abandon the letsencrypt & manual cert management. I think we can just have one prd and one dev ALB for all apps and get the price down plenty.

**Comment by matschaffer** [comment]
I just noticed https://hub.docker.com/r/grafana/grafana-dev/tags?page=1&ordering=last_updated has some sort of "nightly" images, but not sure how up to date they're kept. The tags are all 7.5.0 but the latest release is 7.5.1

**Comment by matschaffer** [comment]
AWS had our old instance slated for replacement so I went ahead with this as-is. We still don't have a good answer to running grafana nightly builds (since they don't provide docker images for them), but we also can't upgrade to 8 at present anyway since the panodata map plugin doesn't run on it.

I merged https://github.com/Safecast/reporting/pull/11, created a new prod env, so I'd say we can call this done.
